after_sign_hook.js

```diff
@@ -1,15 +1,56 @@
 const fs = require('fs');
 const path = require('path');
+const dotenv = require('dotenv');
 
 module.exports = async function (params) {
   // Only notarize the app on Mac OS only.
   if (process.platform !== 'darwin') {
     return;
   }
 
-  if (!process.env.CIRCLE_TAG || process.env.CIRCLE_TAG.length === 0) {
-    console.log('Not on a tag. Skipping notarization'); // eslint-disable-line no-console
-    return;
+  const appStoreConnectKeyPath = path.join(
+    process.env.HOME,
+    '.configure',
+    'simplenote-electron',
+    'secrets',
+    'app_store_connect_api_key.p8'
+  );
+
+  const envPath = path.join(
+    process.env.HOME,
+    '.a8c-apps/simplenote-electron.env'
+  );
+  if (fs.existsSync(envPath)) {
+    dotenv.config({ path: envPath });
+  } else {
+    // eslint-disable-next-line no-console
+    console.log(
+      `No env file found at ${envPath}. Looking for required env vars individually...`
+    );
+    let errors = [];
+    if (process.env.APP_STORE_CONNECT_API_KEY_KEY_ID === undefined) {
+      errors.push(
+        'APP_STORE_CONNECT_API_KEY_KEY_ID value not found in env. Please set it.'
+      );
+    }
+    if (process.env.APP_STORE_CONNECT_API_KEY_ISSUER_ID === undefined) {
+      errors.push(
+        'APP_STORE_CONNECT_API_KEY_ISSUER_ID value not found in env. Please set it.'
+      );
+    }
+    if (fs.existsSync(appStoreConnectKeyPath) === false) {
+      errors.push(
+        `Key file not found at ${appStoreConnectKeyPath}. Please add it.`
+      );
+    }
+
+    if (errors.length > 0) {
+      throw new Error(
+        `Could not begin signing macOS build. Errors: ${errors.join('\n')}`
+      );
+    } else {
+      console.log('All required env vars found. Moving on...');
```

Comment:
Interestingly, we are not getting a lint issue for this one, like the one above 🤷‍♂️